### DISCUSSION TITLE
Office Hours

### DISCUSSION THREAD

**Discussion by yhakbar** [gruntwork-io/terragrunt#3561] [Office Hours Agenda]
Office Hours are a recurring community call that takes place in the [Terragrunt Discord](https://discord.gg/SPu4Degs5f) every other week from 10:30 ET / 14:30 GMT - 11:00 ET / 15:00 GMT.

Maintainers will endeavor to preemptively post the agenda for the call in this GitHub Discussions category so that feedback can be collected, and notes can be tracked after each call.

If you have any feedback on Terragrunt Office Hours, feel free to share!

**Comment by Pulkit12966** [comment]
learning resources for terragrunt
